<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>IP Serveur</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_IPServeur">
             <property name="text">
              <string>172.18.58.150</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="minimum">
              <number>1025</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Texte à envoyer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMessage"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Connexion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Envoyer">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Envoyer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Quitter">
           <property name="text">
            <string>Quitter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Messages et états</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>client</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
